dcrwallet.go

```diff
@@ -322,6 +325,7 @@ func rpcClientConnectLoop(legacyRPCServer *legacyrpc.Server, loader *wallet.Load
 		})
 
 		chainClient.WaitForShutdown()
+		loader.StopTicketPurchase()
```

Comment:
please assign the error to _ so it's explicit that it is being ignored (comment would be nice too)